src/scrollbar.js

```diff
@@ -89,10 +91,6 @@ export default class ScrollBar extends Component {
     }
   }
 
-  updateScroll() {
```

Comment:
This method should be kept for the case:  https://github.com/goldenyz/react-perfect-scrollbar#updatescroll

It is better to implement this method as follows by `onSync`:
```js
updateScroll() {
  if (typeof this.props.onSync === 'function') this.props.onSync(this._ps);
}
```

Reply:
okay. 

BTW, I will update the `README.md`.

Reply:
done, thanks for you review.